### ISSUE TITLE
No restore logic in evaluate.py

### ISSUE THREAD

**Issue by renmengye** [deepmind/learning-to-learn#20]
I trained a MNIST network using `train.py` and saved in `./mnist` folder. 

In `evaluate.py` there is no logic that restores the saved network. 

I printed the loss, it doesn't seem to decrease and I printed deltas, which are exactly the same as doing gradient descent (i.e. LSTM initialized at 0).

**Comment by ko3n1g** [comment]
can you elaborate on the solution? trying to get MNIST to work as well, but no luck so far. My tensorflow version is 1.5, dm-sonnet 1.10, running on GPU, and the mnist-bug (ticket #22) is implemented..
